Source/WebCore/PAL/pal/avfoundation/OutputContext.mm

```diff
@@ -77,16 +76,9 @@
         return [m_context deviceName];
 
     StringBuilder builder;
-    auto devices = outputDevices();
-    auto iterator = devices.begin();
-
-    while (iterator != devices.end()) {
-        builder.append(iterator->name());
-
-        if (++iterator != devices.end())
-            builder.append(" + "_s);
-    }
-
+    builder.append(interleave(outputDevices(), [](auto& device) {
```

Comment:
I don't think we need to explicitly define a `StringBuilder`, just `makeString`.